engine/source/materials/mat/mat090/sigeps90.F

```diff
@@ -86,6 +85,7 @@ SUBROUTINE SIGEPS90(
 #include      "impl1_c.inc"
 C
       INTEGER, INTENT(IN) :: NEL,NUVAR,ISMSTR,ISRATE,IHET
+      INTEGER, INTENT(IN) :: NVARTMP
```

Comment:
VARTMP is not a very descriptive name, could you add a comment in the variable declaration explaining what it contains? This would help in understanding its purpose in the code.